UI/UI.tscn

```diff
@@ -51,17 +59,27 @@ func _on_root_markers_loaded():
 
 func _on_enemy_set_placemark_mouse_entered(enemy_set_placemark: EnemySetPlacemark) -> void:
 	$status_view/container/SetLabel.text = enemy_set_placemark.enemy_set.to_string()
+	_store_placemark(enemy_set_placemark)
 	
 func _on_enemy_set_placemark_mouse_exited(enemy_set_placemark: EnemySetPlacemark) -> void:
 	$status_view/container/SetLabel.text = \"\"
+	set_instance = null
 	
 func _on_gathering_spot_placemark_mouse_entered(gathering_spot_placemark: GatheringSpotPlacemark) -> void:
 	$status_view/container/SetLabel.text = gathering_spot_placemark.gathering_spot.to_string()
+	_store_placemark(gathering_spot_placemark)
 	
 func _on_gathering_spot_placemark_mouse_exited(gathering_spot_placemark: GatheringSpotPlacemark) -> void:
 	$status_view/container/SetLabel.text = \"\"
+	set_instance = null
+	
+func _store_placemark(set_placemark):
+	set_instance = set_placemark
```

Comment:
not a very descriptive function and variable name, maybe something like "hovered_placemark" would be better